docs/mindsdb-docs/docs/setup/custom-config.md

```diff
@@ -0,0 +1,75 @@
+# Custom Configuration of MindsDB Locally
```

Comment:
Can we name this something like `Extend default mindsdb configuration` or similar?

Reply:
Done. Changed the navigation header and title.